misk/src/test/kotlin/misk/moshi/BigDecimalAdapterTest.kt

```diff
@@ -0,0 +1,36 @@
+package misk.moshi
+
+import com.squareup.moshi.Moshi
+import misk.MiskTestingServiceModule
+import misk.inject.KAbstractModule
+import misk.testing.MiskTest
+import misk.testing.MiskTestModule
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import java.math.BigDecimal
+import javax.inject.Inject
+
+@MiskTest(startService = false)
+internal class BigDecimalAdapterTest {
+  @MiskTestModule
+  val module = TestModule()
+
+  @Inject
+  lateinit var moshi: Moshi
+
+  @Test
+  fun `adapter converts from and to json`() {
+    val json = "\"5.5\""
+    val value = BigDecimal("5.5")
+    val jsonAdapter = moshi.adapter<BigDecimal>()
+    assertThat(jsonAdapter.toJson(value)).isEqualTo(json)
+    assertThat(jsonAdapter.fromJson(json)).isEqualTo(value)
+  }
+
+  class TestModule : KAbstractModule() {
+    override fun configure() {
+      install(MiskTestingServiceModule())
+      install(MoshiAdapterModule(BigDecimalAdapter))
```

Comment:
Didn't want to add the BigDecimalAdapter as a default adapter, but open to put it in the default adapters.

Reply:
Why not? I think it’d be great!